ydb/docs/en/core/integrations/hibernate.md

```diff
@@ -0,0 +1,130 @@
+# {{ ydb-short-name }} dialect for Hibernate #
+
+## Overview {#overview}
+
+This is a guide to using Hibernate with YDB.
+
+Hibernate is an Object-Relational Mapping (ORM) framework for Java that facilitates the mapping of object-oriented models. Each database management system (DBMS) partially implements the SQL standard, interpreting certain constructs in its own way.
+
+## Installation {#install-dialect}
+
+Add the following dependency to your project:
+
+{% list tabs %}
+
+- Maven
+
+    ```xml
+    <dependency>
+        <groupId>tech.ydb.dialects</groupId>
+        <artifactId>hibernate-ydb-dialect</artifactId>
+        <!-- Set actual version -->
+        <version>${hibernate.ydb.dialect.version}</version> 
+    </dependency>
+    ```
+
+- Gradle
+
+    ```groovy
+    dependencies {
+        implementation 'tech.ydb.dialects:hibernate-ydb-dialect:$version' // Set actual version
+    }
+    ```
+
+{% endlist %}
+
+If you use Hibernate version 5, you need `<artifactId>hibernate-ydb-dialect-v5</artifactId>`.
```

Comment:
This looks like it belongs to the "Maven" tab above + it needs a Gradle equivalent.